NodeApp/TodoList.js

```diff
@@ -0,0 +1,61 @@
+var express = require("express");
+const fs = require('fs');
+const zlib = require('zlib');
+var app = express();
+let idIndex = 5;
+
+// Method to show contents of the list to the user:
+app.get("/View", (req, res) => {
+    let jsonFile = fs.readFileSync('./MyFile.json');
+    let file = JSON.parse(jsonFile);
+    res.send(file);
+})
+
+// Method for adding new items to list:
+app.post("/Add", (req, res) => {
```

Comment:
the /new is redundant.
restful api should be using a single controller with the different method types e.g:
GET /tasks should return all tasks
GET /tasks?id=1 should return task with id 1
POST /tasks should insert a new task
DELETE /tasks?id=1 should delete task with id 1
etc..